runtimes/huggingface/tests/test_tasks/test_japanese_mask_fill.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from mlserver.settings import ModelSettings, ModelParameters
+from mlserver_huggingface import HuggingFaceRuntime
+from mlserver_huggingface.codecs import HuggingfaceRequestCodec
+
+
+@pytest.fixture
+def bert_japanese_settings(request) -> ModelSettings:
+    if (not request.config.option.test_hg_tasks) or (
+        not request.config.option.test_ja_support
+    ):
+        pytest.skip()
+
+    return ModelSettings(
+        name="model",
+        implementation=HuggingFaceRuntime,
+        parameters=ModelParameters(
+            extra={
+                "task": "fill-mask",
+                "pretrained_model": "cl-tohoku/bert-base-japanese",
+                "pretrained_tokenizer": "cl-tohoku/bert-base-japanese",
+            }
+        ),
+    )
+
+
+@pytest.fixture
+async def current_runtime(bert_japanese_settings: ModelSettings) -> HuggingFaceRuntime:
+    runtime = HuggingFaceRuntime(bert_japanese_settings)
+    await runtime.load()
+    return runtime
+
+
+@pytest.fixture
+def japanese_text_request():
+    # Test sentence: Is the sky really [MASK]?
+    test_sentence = "実際に空が[MASK]のか？"
+
+    inputs = [test_sentence]
+    req = HuggingfaceRequestCodec.encode_request(
+        {"inputs": inputs},
+        use_bytes=False,
+    )
+    return req
+
+
+async def test_infer(current_runtime, japanese_text_request):
```

Comment:
I think this test function should be parametrised with test cases [(input_request, output_request),...] even though you just have one at the moment to allow for future extensions.

Reply:
Hey @sakoush , I think all of your comments are addressed. I appreciate a review when you get the chance. Enjoy your holidays!